phoenix-tools/src/main/java/org/apache/phoenix/schema/SchemaExtractionProcessor.java

```diff
@@ -54,9 +55,6 @@ public String process() throws Exception {
         if (ddl != null) {
             return ddl;
         }
-        if(this.table.getColumnFamilies().size() > 1 ) {
```

Comment:
This PR does not completely support the multiple column families, especially if there are different properties for each column family as mentioned by Goeffrey on the previous review. Let's modify this if clause to be specific unless you want to include the change in this one and completely get rid of "Not Supported Exception".